typescript/infra/scripts/warp-routes/monitor/monitor-warp-route-balances.ts

```diff
@@ -310,6 +322,48 @@ async function getCoinGeckoApiKey(): Promise<string | undefined> {
   return apiKey;
 }
 
+function getWarpRouteCollateralTokenSymbol(warpCore: WarpCore): string {
+  // We need to have a deterministic way to determine the symbol of the warp route
+  // as its used to identify the warp route in metrics. This method should support routes where:
+  // - All tokens have the same symbol, token standards can be all collateral, all synthetic or a mix)
+  // - All tokens have different symbol, but there is a collateral token to break the tie, where there are multiple collateral tokens, alphabetically first is chosen
+  // doesn't support routes where:
+  // - All tokens have different symbols and there is no collateral token to break the tie
```

Comment:
any thoughts on just arbitrarily picking the first alphabetical one?

I don't feel strongly though, this feels unlikely to be a case we hit and I'm happy to deal with it later if needed